tests/fixtures/typescript-types

```diff
@@ -1,8 +1,10 @@
 {number}
 {{ object: with, comma: separator }}
 {{ object: with, trailing: comma, }}
-// {{ object: with\n linebreak: separator }}
-// {{ object: with\n linebreak: separators\n }}
+{{ object: with; semicolon: separator }}
+{{ object: with; semicolon: separators; }}
+{{ object: with \n linebreak: separator }}
```

Comment:
Some tests with `\t`, `\r` or ` ` would be nice here.

Reply:
I'll wait on your reply above to have an idea of the exact types of tests to add here. I expect you'd want something like the following for passing:
```
{{ object: with \r\n carriageReturnAndLinebreak: separator }}
{{ object: with \t\n tabAndLinebreak: separator }}
```
and these for rejecting:

```
{{ object: with \t tab: separator }}
{{ object: with \r\t carriageReturnAndTab: separator }}
```

Reply:
Those are good. Here's a few more:

```
\n\r <-- reject (I think?)
\t\r\t\r\n <-- reject
\r <- reject
```

Reply:
Nice--I've also added an additional rejecting one with two carriage returns in a row followed by a newline.